libdino/src/service/muc_manager.vala

```diff
@@ -285,6 +308,9 @@ public class MucManager : StreamInteractionModule, Object {
         stream_interactor.module_manager.get_module(account, Xep.Muc.Module.IDENTITY).invite_received.connect( (stream, room_jid, from_jid, password, reason) => {
             invite_received(account, room_jid, from_jid, password, reason);
         });
+        stream_interactor.module_manager.get_module(account, Xep.Muc.Module.IDENTITY).voice_request_received.connect( (stream, room_jid, from_jid, nick, role, label) => {
```

Comment:
this line (and others below) are very long, i suggest to wrap. I think it is better when readers of Github code don't have to use side-scroll.

Reply:
These lines are written in a similar fashion as other lines in `on_account_added` method. I just adhered to the codebase convention.